--- src/Makefile.org	Sat Aug  7 12:35:13 1999
+++ src/Makefile	Thu Feb 24 12:57:48 2000
@@ -21,7 +21,7 @@
 # Use an ANSI compiler (or at least one that groks prototypes and void *):
 CC=gcc
 ##CFLAGS=-ggdb -DWITH_LOCKING -DWITH_TMPFILE -DWITH_VARSUP -O2 -m486 -fomit-frame-pointer -DSYSVR4
-CFLAGS = -ggdb -DWITH_LOCKING -DWITH_TMPFILE -DWITH_VARSUP -DWITH_EXPR -DWITH_MISC -O2 -m486 -DSYSVR4
+CFLAGS = -ggdb -DWITH_LOCKING -DWITH_TMPFILE -DWITH_VARSUP -DWITH_EXPR -DWITH_MISC -DSYSVR4 $(RPM_OPT_FLAGS)
 LDFLAGS = libedit.a
 
 # You may substitute "bison -y" for yacc. (You want to choose the one that
